src/types/service_config.go

```diff
@@ -13,5 +13,31 @@ type ServiceConfig struct {
 	ServiceMessages `yaml:"messages,omitempty"`
 	Docker          DockerConfig       `yaml:",omitempty"`
 	Dependencies    []DependencyConfig `yaml:",omitempty"`
+	Environment     envVars            `yaml:",omitempty"`
 	Production      map[string]string  `yaml:",omitempty"`
 }
+
+type envVars struct {
+	Default     map[string]string `yaml:",omitempty"`
+	Development map[string]string `yaml:",omitempty"`
+	Production  map[string]string `yaml:",omitempty"`
+	Secrets     []string          `yaml:",omitempty"`
+}
```

Comment:
What if it's not being used outside of this file? This seems more like a sub-struct of `ServiceConfig`.